legacy/src/app/partials/intro-user.html

```diff
@@ -2,10 +2,10 @@
     <div class="row">
         <div class="p-card--highlighted">
             <div class="p-card__content u-no-margin--top">
-                <h2 class="p-card__title p-heading--four">
+                <h4 class="p-card__title">
```

Comment:
Driveby fixes for icons in titles - icon only matches heading size according to the tag, not the class. The images partial is used in the images page (obviously) as well as the intro page, with the only differences being these heading have icons and the page is wrapped in a highlighted card instead of a plain div.